app.py

```diff
@@ -62,3 +62,15 @@ def path():
 def test_database():
     logger.info("Reached /api/test-database")
     return jsonify(db.get_all_test())
+
+@app.route("/search", methods=["GET", "POST"])
+def search():
+    if request.method == "GET":
+        query = request.args.get("query")
```

Comment:
this returns a string. to convert import json and do json.loads(request.args.get("query")). this should convert it to a json object